### PR TITLE
Added new experiment which runs traffic replay

### PR DESCRIPTION
**Pull request by ghost**
Simple experiment which runs replay and creates jitter result files(not sending to server yet). Next commit will parse these results into a json and upload to the server like other experiments.

### CODE REVIEW

File: python_lib.py [hunk 1]
```diff
@@ -0,0 +1,657 @@
+import sys, os, ConfigParser, math, json, time, subprocess, commands, random, string, logging, logging.handlers, socket, StringIO
```
Comment (rpanah) [inline python_lib.py]: This whole file is still in the middle of the repository, and has a vague name. This was supposed be merged with the experiment primitive file and placed in the primitives.

File: replay_client.py [hunk 2]
```diff
@@ -0,0 +1,841 @@
+'''
```
Comment (rpanah) [inline replay_client.py]: This file too, is in the middle of the repository. This has to be formatted as a primitive and placed in one directory with the rest of them (`centinel/primitives`).
Additionally, this hasn't been cleaned at all. I still see a lot of debug flags and other unused parameters that haven't been cleaned up.

File: python_lib.py [hunk 3]
```diff
@@ -0,0 +1,657 @@
...
+    pass
+
+def name2code(key, what):
+    '''
+    Codes are 5 digits and follow these rules (just for the hech of it!):
+        right most: 1 if random, 0 otherwise
+        2 left most: app code
+        2 midd: subapp code
+    '''
+    names = {
```
Comment (rpanah) [inline python_lib.py]: These are still hard-coded. Weren't they supposed to a configuration file?

File: python_lib.py (hunk 1 above)
Comment (spaduru) [inline python_lib.py]: Arash used this file to pickle and unpickle objects and the namespace needs to match to unpickle the current Pickle files. I removed unnecessary code except the objects that are being used. Once we create new pickle objects, this file can be deleted.

File: python_lib.py (hunk 3 above)
Comment (spaduru) [inline python_lib.py]: Removed.

**Comment by rpanah** [comment]
Remove old pickle files. Record new ones (you might need to consult Arash regarding how/where to do this).

**Pull request closed without merging**